cgo/registry.go

```diff
@@ -0,0 +1,49 @@
+package cgo
+
+import (
+	"context"
+	"sync"
+)
+
+var (
+	mu       sync.RWMutex
```

Comment:
@Stebalien are you planning on pulling the atomic version into this?

Reply:
I meant this one that you wrote: https://github.com/filecoin-project/cgo-blockstore/blob/master/blockstore.go#L17

Reply:
Oh, hell no. Not unless this turns out to be the bottleneck.